units/crons_remote_minute.go

```diff
@@ -78,11 +77,12 @@ func (j *cronsRemoteMinuteJob) Run(ctx context.Context) {
 	catcher.Wrap(amboy.EnqueueManyUniqueJobs(ctx, j.env.RemoteQueue(), allJobs), "populating main queue")
 	catcher.Add(enqueueHostSetupJobs(ctx, j.env, j.env.RemoteQueue(), ts))
 
-	// Create dedicated queues for host creation, event notifier, and commit queue jobs.
+	// Create dedicated queues for host creation, host termination, event notifier, and commit queue jobs.
```

Comment:
We can remove this comment because 1. it's already missing the spawn host modification queue group in the comment (so it's not updated) and 2. it's just restating what the code is doing.

Reply:
I think this comment got missed? I don't care that much if we want to keep it around, but maybe got lost in the many `require.NoError` comments.

Reply:
ah yes i did miss this thanks!